File: daily/C++版本/349. 两个数组的交集.cpp
class Solution {
public:
    vector<int> intersection(vector<int>& nums1, vector<int>& nums2) {
        vector<int> vec,res;
        for(int i=0;i<nums1.size();i++)
            for(int j=0;j<nums2.size();j++)
                if(nums1[i]==nums2[j])
                    vec.push_back(nums1[i]);
                    
        set<int> s(vec.begin(),vec.end());
        for(set<int>::iterator temp=s.begin();temp!=s.end();temp++)
            res.push_back(*temp);
        return res;
    }
};

File: daily/二分查找/二分查找easy/349. 两个数组的交集.js
/**
 * @param {number[]} nums1
 * @param {number[]} nums2
 * @return {number[]}
 */
var intersection = function(nums1, nums2) {
    var res=[];
    for(let i=0;i<nums1.length;i++){
        for(let j=0;j<nums2.length;j++){
            var flag=res.indexOf(nums1[i]);
            if(flag==-1 && nums1[i]==nums2[j])
                res.push(nums1[i]);
        }
    }
    return res;
};
